### PR TITLE
Clean up exam and group pages

### LINKED ISSUE #1935: alternate 'exam visibility' handling for classes with no groups?
For classes with no groups, the 'visible to' UI is a bit confusing:

----

image

----

image

----

The user is presented with an empty dropdown and a single radio button. It might be good to add some special-case informational text for users that are unfamiliar with groups.

### LINKED ISSUE #2046: Groups page without any groups
## Summary

* Currently, if there are no groups created, ungrouped users can still be selected but no action can be performed. Looking for a mockup for this scenario.

whattodo

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

* Fixes #2046
* Fixes #1935
* Right align modal buttons.
* Wording update to exam  visibility modal
* Exam row changes as seen in https://projects.invisionapp.com/share/7D8ZDY4EN#/screens/242551478
* Replace radios with filter.
 
127 0 0 1-8000-coach-

127 0 0 1-8000-coach- 1

127 0 0 1-8000-coach- 2

### CODE REVIEW

**Comment by indirectlylit** [comment]
should we target this to develop?

File: kolibri/plugins/coach/assets/src/views/exams-page/change-exam-visibility-modal.vue [hunk 1]
```diff
@@ -38,10 +39,9 @@
   export default {
     name: 'changeExamVisibilityModal',
     $trs: {
-      examVisibility: 'Exam visibility',
-      shouldBeVisible: "'{ examTitle }' should be visible to:",
+      examVisibility: 'Change exam visibility',
+      shouldBeVisible: `Make '{ examTitle }' visible to entire class or specific groups`,
```
Comment (jonboiser) [inline kolibri/plugins/coach/assets/src/views/exams-page/change-exam-visibility-modal.vue]: Using backticks without string interpolation looks kind of weird. Is it critical that `examTitle` be in single quotes? You could also do the whole message in single quotes, but escape the quotes in the middle `'...\'{examTitle}\'..'`

Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/exams-page/change-exam-visibility-modal.vue]: This is something that should be formatted by `prettier`.

In the past, I've seen it use the strategy shown in the removed line - wrapping with double quotes:

```javascript
"Make '{ examTitle }' visible to entire class or specific groups"
```

Comment (jonboiser) [reply kolibri/plugins/coach/assets/src/views/exams-page/change-exam-visibility-modal.vue]: Yeah, maybe go back to using double-quotes for the whole thing. Prettier will not correct using unnecessary backticks (certain ESLint configs would, though). But prettier will actually autoformat 'hello i said 'hello'' to "hello i said 'hello'" (i.e. this is how it prefers to handle single-quotes).

**Comment by jonboiser** [comment]
Code looks good.

* Tested out fixes for #2046 and #1935
* Looked at all the modals
* Looked at all changes mentioned in OP

The only minor thing is the typography of table headers on the Exam page. Maybe you could copy paste some styles from Manage Content Page or Users page.

**Comment by christianmemije** [comment]
@jonboiser Updated quotes and table header.

**Pull request merged**